web/static/js/app.jsx

```diff
@@ -957,9 +953,11 @@ var Markdown = React.createClass({
     markedRenderer.image = this.replaceRelativeUrlWithGithubImgSrc;
     var markedHtml = marked(this.props.children || '',
       {renderer: markedRenderer});
+    /* eslint-disable react/no-danger */
```

Comment:
This seems like a pretty silly lint rule to me; the prop name should be enough to discourage people. Maybe we can disable it?